requirements.txt

```diff
@@ -1,3 +1,6 @@
-asgiref==3.6.0
-Django==4.1.7
-sqlparse==0.4.3
+asgiref==3.6.0
+Django==4.1.7
+sqlparse==0.4.3
+bs4==0.0.1

```

Comment:
Why this version? This is very outdated. There is a major version 4 we should use that.

Reply:
The bs4 library isn't the same as BeutifulSoup library

Reply:
We shouldn't use the dummy wrapper for this library if we can use the official one.

Reply:
Sorry for the confusion, the version of bs4 isn't the same as version of BeautifulSoup